libs/application/ui-shell/src/components/Screen.tsx

```diff
@@ -239,6 +254,77 @@ const Screen: FC<ScreenProps> = ({
   const { width } = useWindowSize()
   const headerHeight = 85
 
+  const resetUnionValues = useCallback(async () => {
```

Comment:
Original ticket was to make union more explicit, so now it follows the approach from privatePensionFund, where u have radio button and u have to explicitly choose yes or no.

But Darri found a bug:
<img width="786" alt="image" src="https://user-images.githubusercontent.com/26793642/158828795-349b1fb9-0065-4bc6-afe4-a1d455b20c23.png">

TLDR: when u click yes, fill in the fields and then click no, u are still submitting fields that shouldn't be there.

I know that this solution is bad workaround, but I tried few others and I can't make it work, because of the app "frameworky" structure and the fact that part of the data is coming from api, part of the data is stored in local state and all of that has to be synced together. If u have better idea I am all ears or let me know if u need more info.